tests/unit_tests/test_amazon_bedrock_embedding_function.py

```
@@ -0,0 +1,458 @@
+    """
+    if not is_boto3_available():
+        return False
+
+    credentials = None
+    try:
+        import boto3
+
+        # Try to create a session and check if credentials are available
+        session = boto3.Session()
+        credentials = session.get_credentials()
+    except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)